src/components/CartPage/Order.jsx

```diff
@@ -0,0 +1,111 @@
+import { DefaultButton, DivideUnderLine } from 'components/shared/styles';
+import styled from 'styled-components';
+import PropType from 'prop-types';
+
+function Order({ checkedList }) {
+  const getExpectedPrice = () =>
+    checkedList.reduce(
+      (sum, current) => sum + Number(current.price) * current.quantity,
+      0,
+    );
```

Comment:
아래 코드로 하면 어떤 차이가 있을까요? 어떤 게 더 나을까요!?
```suggestion
  const expectedPrice = checkedList.reduce(
    (sum, current) => sum + Number(current.price) * current.quantity,
  0);
```

Reply:
제가 한 것은 함수를 정의해놓고 사용하는 부분에서 함수를 호출해서 사용을 하고 월터님이 제안해주신 코드는 컴포넌트가 실행될때 즉시 계산을 해서 값을 넣어주는 코드네요.

어차피 함수 컴포넌트가 실행될때마다 계속해서 계산돼야하는 값이기 때문에 함수를 정의하고 다시 호출하는 불필요한 과정보다는 즉시 값을 계산해서 변수에 담아두고 해당 변수를 사용하는 것이 더 설득력있어보입니다.

혹시 제가 캐치하지 못한 월터님의 의도가 있을까요?

Reply:
제안해주신 코드로 수정했습니다!

[해당커밋](https://github.com/woowacourse/react-shopping-cart/pull/142/commits/8e114b78a2da03eaabfefdbe21d0ba6a6442b79b)